src/Mvc/Mvc.ViewFeatures/src/Rendering/IHtmlHelper.cs

```diff
@@ -375,7 +375,9 @@ public interface IHtmlHelper
     /// <typeparam name="TEnum">Type to generate a select list for.</typeparam>
     /// <returns>
     /// An <see cref="IEnumerable{SelectListItem}"/> containing the select list for the given
-    /// <typeparamref name="TEnum"/>.
+    /// <typeparamref name="TEnum"/>,
+    /// with a decimal representation of the ordinal as <see cref="SelectListItem#Value" />
```

Comment:
Looks like a # isn't supported here. Did you mean to go:

```suggestion
    /// with a decimal representation of the ordinal as <see cref="SelectListItem.Value" />
```